### DISCUSSION TITLE
QZ Zwift Heart Rate Monitor Pairing

### DISCUSSION THREAD

**Discussion by brumohr** [cagnulein/qdomyos-zwift#617] [Q&A] [answered]
I'm running QZ - qdomyos - zwift (ver 2.10.6) on iPad IOS 15.2 and zwift on a PC Win 11, with QZ paired to a Technogym Myrun.

I have been unable to pair either my tickr hrm nor my Apple Watch to QZ.  I would like to know if it is possible and if the HRM can simultaneously be paired to zwift.

Is there a "how-to" published somewhere?

**Comment by cagnulein** [comment]
For apple watch you need to:
- be sure that the Heart Belt Name setting in QZ is "Disabled"
- open the QZ app on the AW
- giving ALL the permission to Apple Health
- press start on the QZ app on the AW
- the Heart value on QZ on the iphone will magically appear

For the tickr, instead:
- set the Heart Belt Name with the tickr name and restart QZ

There are some informations in the wiki about this
Let me know

**Reply by brumohr** [reply]
Followed the instructions for tickr below and it works great!  Thank you Roberto.

 "For the tickr, instead: set the Heart Belt Name with the tickr name and restart QZ"

**Comment by brumohr** [comment, marked as answer]
My problem with AW heart rate not showing on QZ app running on iPad has to do with (I think) the fact that iPad does not support Apple Health.  When using iPhone to run QZ app, AW heart rate sync's up with QZ nicely!  So no issues with QZ app.  Sorry.

So, once again I have gotten bitten by the fact that iPad does not support Apple Health.  What a PITA!
